app/src/main/java/de/danoeh/antennapod/fragment/preferences/StatisticsFragment.java

```diff
@@ -67,7 +67,9 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
     @Override
     public void onStart() {
         super.onStart();
-        ((PreferenceActivity) getActivity()).getSupportActionBar().setTitle(R.string.statistics_label);
+        if (getActivity().getClass() == PreferenceActivity.class) {
+            ((PreferenceActivity) getActivity()).getSupportActionBar().setTitle(R.string.statistics_label);
+        }
```

Comment:
Please do a similar thing to the toolbar. Currently, the screen hides its toolbar but when calling from MainActivity, we need one. See this line in the same file:

```
rootView.findViewById(R.id.toolbar).setVisibility(View.GONE);
```